common/src/test/scala/uk/ac/wellcome/test/utils/SNSLocal.scala

```diff
@@ -17,47 +18,64 @@ trait SNSLocal extends Suite with BeforeAndAfterEach {
       new EndpointConfiguration(localSNSEndpointUrl, "local"))
     .build()
 
-  private val topicName = "es_ingest"
+  private val ingestTopicName = "es_ingest"
+  private val idMinterTopicName = "id_minter"
 
-  //we use this implementation of SNS running in a docker container https://github.com/elruwen/fake_sns/tree/feature-fixed-account-id
-  val ingestTopicArn = amazonSNS.createTopic(topicName).getTopicArn
+  val ingestTopicArn = amazonSNS.createTopic(ingestTopicName).getTopicArn
+  val idMinterTopicArn = amazonSNS.createTopic(idMinterTopicName).getTopicArn
 
   override def beforeEach(): Unit = {
     super.beforeEach()
-    new DefaultHttpClient().execute(new HttpDelete(s"$localSNSEndpointUrl/messages"))
+    new DefaultHttpClient()
+      .execute(new HttpDelete(s"$localSNSEndpointUrl/messages"))
   }
 
+
+  //TODO this is getting a bit complicated.
+  // Try using https://hub.docker.com/r/s12v/sns/ docker image instead of the current one and subscribe a sqs queue to it to check for messages.
+  // Couldn't make it work but should retry without proxies in local development machine
   def listMessagesReceivedFromSNS(): List[MessageInfo] = {
     /*
-    This is a sample returned by the fake-sns implementation:
-    ---
-    topics:
-    - arn: arn:aws:sns:us-east-1:123456789012:es_ingest
-      name: es_ingest
-    subscriptions: []
-    messages:
-    - :id: 1a9f3f37-54be-4a1e-9798-7fb928caad87
-      :subject: subject
-      :message: '"some" "Message"'
-      :topic_arn: arn:aws:sns:us-east-1:123456789012:es_ingest
-      :structure:
-      :target_arn:
-      :received_at: 2017-04-10 16:31:38.015426145 +00:00
-
+This is a sample returned by the fake-sns implementation:
+---
+topics:
+- arn: arn:aws:sns:us-east-1:123456789012:es_ingest
+  name: es_ingest
+- arn: arn:aws:sns:us-east-1:123456789012:id_minter
+  name: id_minter
+messages:
+- :id: acbca1e1-e3c5-4c74-86af-06a9418e8fe4
+  :subject: Foo
+  :message: '{"identifiers":[{"source":"Miro","sourceId":"MiroID","value":"1234"}],"title":"some
+    image title","accessStatus":null}'
+  :topic_arn: arn:aws:sns:us-east-1:123456789012:id_minter
+  :structure:
+  :target_arn:
+  :received_at: 2017-04-18 13:20:45.289912607 +00:00
      */
 
     val string = scala.io.Source.fromURL(localSNSEndpointUrl).mkString
-    string
-      .substring(string.indexOf("- :id:") + 1)
-      .split("\n- ")
-      .map { messageDetails =>
-        val messageLines = messageDetails.split('\n')
-        MessageInfo(
-          getMessageLine(messageLines, ":id: "),
-          getMessageLine(messageLines, ":message: ").replace("'", ""),
-          getMessageLine(messageLines, ":subject: "))
-      }
-      .toList
+    debug(s"""Messages received by fake-sns:
```

Comment:
Yes, there's a TODO at the beginning of the function about switching image. I think for now this message makes sense as it is as when we change docker image this whole function is going to disappear